src/networks/mod.rs

```diff
@@ -453,6 +455,20 @@ macro_rules! make_height {
     };
 }
 
+// The formula matches lotus
+// ```go
+// sinceGenesis := build.Clock.Now().Sub(genesisTime)
+// expectedHeight := int64(sinceGenesis.Seconds()) / int64(build.BlockDelaySecs)
+// ```
+// See <https://github.com/filecoin-project/lotus/blob/b27c861485695d3f5bb92bcb281abc95f4d90fb6/chain/sync.go#L180>
+pub fn calculate_expected_epoch(
```

Comment:
I'm partial to leaving out `calculate_` from function names. But I'm fine either way.